source/includes/examples/_simpy-elevator.html.md

```diff
@@ -18,87 +18,59 @@ For more ideas of how SimPy can simulate real world problems see the [SimPy Exam
 
 ## Inkling File
 
-###### Schema
-
-```inkling
-# Position is current location of elevator
-# State of each floor: 1 if the floor is requested, 0 if not
-schema FloorState
-    Int8{0, 1, 2} Position,
-    Int8{0, 1} Floor1,
-    Int8{0, 1} Floor2,
-    Int8{0, 1} Floor3
-end
+###### Types
+
+```inkling2
+type FloorState {
+    Position: number<0, 1, 2>,
+    Floor1: number<0, 1>,
+    Floor2: number<0, 1>,
+    Floor3: number<0, 1>
+}
 ```
 
-The `FloorState` schema defines the dictionary returned from the Python simulation's `advance` method to the BRAIN.
+The `FloorState` type defines the fields returned by the simulation's `advance` method.
 
-```inkling
-# command options: up, open, down
-constant Int8 up = 0
-constant Int8 open = 1
-constant Int8 down = 2
-schema Action
-    Int8{up, open, down} command
-end
+```inkling2
+type Action {
+    command: number<up = 0, open = 1, down = 2>
+}
 ```
 
-The `Action` schema defines the possible actions the elevator can take. We have added constants to this schema to demonstrate how they can be optionally used. In this case the command given to the elevator is '0' is open, '1' is go up a floor, and '2' is go down a floor.
+The `Action` type defines the possible commands accepted by the elevator. In this case, the command given to the elevator is '0' to open the door, '1' to go up a floor, and '2' to go down a floor.
 
-```inkling
-# Possible option for configuration
-schema ElevatorConfig
-    Int8 episode_length
-end
+```inkling2
+type ElevatorConfig {
+    episode_length: number
```

Comment:
I'd use `Number.Int8` -- floating point episode_lengths don't make sense, and the engine picks random values in a given range for a constraint...